### PR TITLE
Fix incomplete timestamp support and wait_for_valid_timestamp not ret…

### LINKED ISSUE #591: Incomplete timestamp support - missing attributes and wait_for_valid_timestamp isn't returning a value
In the round of updates for 1.0, we missed a batch of attributes and functionality related to timestamps. In particular these attributes listed in [attribute_helpers.py exclusion list](https://github.com/ni/nidaqmx-python/blob/master/src/codegen/utilities/attribute_helpers.py) should be supported:

```python
    "ARM_START_TRIG_TIMESTAMP_VAL",
    "REF_TRIG_TIMESTAMP_VAL",
    "START_TRIG_TIMESTAMP_VAL",
    "FIRST_SAMP_CLK_OFFSET",
    "FIRST_SAMP_CLK_TIMESCALE",
    "FIRST_SAMP_CLK_WHEN",
    "FIRST_SAMP_TIMESTAMP_VAL",
    "SYNC_PULSE_TIME_WHEN",
    "TIMING_SYNC_PULSE_FORCE", # Note: not this one, this is unrelated and isn't actually supported by any DAQmx devices. Strange.
    "ARM_START_TRIG_TIMESTAMP_VAL",
    "REF_TRIG_TIMESTAMP_VAL",
    "START_TRIG_TIMESTAMP_VAL",
```

While investigating this, @bkeryan noticed a similar issue that is related - [`task.wait_for_valid_timestamp` isn't returning a value](https://github.com/ni/nidaqmx-python/blob/cc140f75e3ba7ab6c0486e8e12e117ac0a7809c9/src/handwritten/task/_task.py#L1024). Both the [gRPC interpreter](https://github.com/ni/nidaqmx-python/blob/cc140f75e3ba7ab6c0486e8e12e117ac0a7809c9/generated/nidaqmx/_grpc_interpreter.py#L3333) and the [library interpreter](https://github.com/ni/nidaqmx-python/blob/cc140f75e3ba7ab6c0486e8e12e117ac0a7809c9/generated/nidaqmx/_library_interpreter.py#L5888) are correctly returning a datetime object.

### PR DESCRIPTION
**Pull request by WayneDroid**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nidaqmx-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- removed timestamp attributes from exclusion list.
- return timestamp value from wait_for_valid_timestamp().
- updated unit test cases

### Why should this Pull Request be merged?

Closes #591 

### What testing has been done?

run `poetry run pytest tests/ -q` - pass

image

### CODE REVIEW

**Review by zhindes — APPROVED** [review]

**Comment by zhindes** [comment]
FYI - I updated your PR description to use the [closes keyword](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) for the issue!

File: tests/component/task/test_triggers.py [hunk 1]
```diff
@@ -91,7 +91,8 @@ def test___start_trigger___wait_for_valid_timestamp___no_errors(
     ai_voltage_task.triggers.start_trigger.timestamp_enable = True
     ai_voltage_task.start()
 
-    ai_voltage_task.wait_for_valid_timestamp(TimestampEvent.START_TRIGGER)
+    timestamp = ai_voltage_task.wait_for_valid_timestamp(TimestampEvent.START_TRIGGER)
+    assert isinstance(timestamp, ht_datetime)
```
Comment (bkeryan) [inline tests/component/task/test_triggers.py:95]: [style] Leave a blank line between the act and assert sections of the test

**Review by bkeryan — APPROVED** [review]
Approved with suggestions

File: tests/component/task/test_triggers.py (hunk 1 above)
Comment (WayneDroid) [reply tests/component/task/test_triggers.py]: noted, blank line is added between act and assert for the test cases.

**Pull request merged**